tests/unit/NetworkTest.js

```diff
@@ -478,28 +420,6 @@ test('test bad response will log alert', () => {
         });
 });
 
-test('test Failed to fetch error for non-retryable requests resolve with unable to retry jsonCode', () => {
```

Comment:
Same here. I think this would still be relevant?

Reply:
No because there shouldn't be any non-retryable requests in the sequential queue.